x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/index_properties/helpers.test.ts

```diff
@@ -54,30 +39,27 @@ describe('helpers', () => {
     });
 
     describe('when `mappingsProperties` is unknown', () => {
+      const incompatibleFieldMetadata = {
+        ...EcsFlatTyped['@timestamp'],
+        hasEcsMetadata: true,
+        indexFieldName: '@timestamp',
+        indexFieldType: '-',
+        indexInvalidValues: [],
+        isEcsCompliant: false,
+        isInSameFamily: false,
+      };
       const expected = {
-        all: [],
+        all: [incompatibleFieldMetadata],
         custom: [],
         ecsCompliant: [],
-        incompatible: [
-          {
-            description:
-              'Date/time when the event originated. This is the date/time extracted from the event, typically representing when the event was generated by the source. If the event source has no original timestamp, this value is typically populated by the first time the event was received by the pipeline. Required field for all events.',
-            hasEcsMetadata: true,
-            indexFieldName: '@timestamp',
-            indexFieldType: '-',
-            indexInvalidValues: [],
-            isEcsCompliant: false,
-            isInSameFamily: false,
-            type: 'date',
-          },
-        ],
+        incompatible: [incompatibleFieldMetadata],
```

Comment:
For reviewers:

Per logic of `getPartitionedFieldMetadata` that is used within `getSortedFieldMetadata`, `all` is should always include `incompatible` and the rest